content/benefits-pay-perks/benefits-perks/parental-leave.md

```diff
@@ -0,0 +1,27 @@
+# Parental Leave at Sourcegraph
+
+## Parental leave benefit
+
+Sourcegraph ensures you are paid 100% of your salary for up to 16 weeks (covering the difference between the amount your state parental leave covers and your full salary).
+You may take more or less than 16 weeks, depending on your needs and preferences (anything over 16 weeks is unpaid).
+You may take the 16 weeks consecutively, or you may choose to use them non-consecutively (so long as it is within 12 months of the delivery date).
+
+## Communicating and logging your parental leave
+
+### Notify the People team
+
+Notify People Ops via email (people-ops@sourcegraph.com) as soon as you can (no later than 1 month prior to your leave date) with answers to the following questions:
+What date are you planning on starting your leave?
+What is the due date?
+Are you the birthing caregiver or non-birthing caregiver?
+Which country and state will you be based in during your leave?
+Do you plan on taking 16 weeks (or more/less)?
+Will you be taking your leave consecutively or non-consecutively?
+
```

Comment:
@mercadon What are your thoughts on adding a question: "Are there any questions or concerns you have for the People Ops team regarding your leave?" or something like "Is there any additional support or help you'd like from the People Team?"

Trying to think through a question to pre-emptively answer any questions they might have while making them feel supported.

Reply:
Let's do it! It'll be good data for us too as we build this out more.